src/sharedServices/zendesk/validateZendeskRequest.ts

```diff
@@ -151,9 +151,9 @@ const getTodayUtc = (): number => {
   return Date.UTC(today.getFullYear(), today.getMonth(), today.getDate())
 }
 
-const mapSpaceSeparatedStringToList = (input: string) => {
+const mapSpaceSeparatedStringToList = (input: string): string[] => {
```

Comment:
Don't know if Git will take care of this, but since I merged my PR I moved this function into the `helpers.ts` file, (and moved this file into the `lambdas/initiateDataRequest` folder)

Reply:
Yeah, I think I've maybe got that now after a merge, I'll check